sdk/nodejs/cmd/pulumi-language-nodejs/main.go

```diff
@@ -1378,7 +1378,7 @@ func (host *nodeLanguageHost) RunPlugin(
 		return err
 	}
 
-	env := os.Environ()
+	env := req.Env
```

Comment:
This might need to be append'd

Reply:
Go and Python also just use `req.Env` :thinking: I modeled this after that.

Reply:
we probably need tests to check and enforce this